code/controllers/mc/master.dm

```diff
@@ -124,15 +124,21 @@ var/CURRENT_TICKLIMIT = TICK_LIMIT_RUNNING
 // 	Make a subsystem, give it the SS_NO_FIRE flag, and do your work in it's Initialize()
 /datum/controller/master/proc/Setup()
 	set waitfor = FALSE
-	sleep(1 SECONDS)
-	to_chat(world, "<span class='boldannounce'>Initializing subsystems...</span>")
-
+	sleep(-1)
+	//moving this random bullshit into here, because it didn't belong in world/New()
+	//SetupHooks() // /N3X15 project from 8 years ago (WIP)
+	createDatacore()
```

Comment:
why are these procs procs? Why do they return one? Why not put the contents of those procs here directly?

Reply:
I'd plan to put them in a subsystem eventually, much is likely more appropriate, then that way they can be removed and the SS init will handle them properly